app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save
```

Comment:
Do we need to bulk this up in regards to edge cases?

Reply:
That's a great point Lisa! We can definitely keep an eye on this for future tests :)